tests/utils/dockercli/volumelifecycle.go

```diff
@@ -144,3 +144,14 @@ func StopContainer(ip, containerName string) (string, error) {
 	log.Printf("Stopping container [%s] on VM [%s]", containerName, ip)
 	return ssh.InvokeCommand(ip, dockercli.StopContainer+containerName)
 }
+
+// IsContainerExist - return true if container exists otherwise false
+func IsContainerExist(ip, containerName string) bool {
+	log.Printf("Checking container [%s] presence on VM [%s]", containerName, ip)
```

Comment:
Use gocheck Logf() instead? May be create a logging function in misc.go and use that in all test code.